### PR TITLE
Add tri samples

### PR DESCRIPTION
**Pull request by guohao**

### CODE REVIEW

File: dubbo-samples-triple/pom.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
     <properties>
         <source.level>1.8</source.level>
         <target.level>1.8</target.level>
-        <dubbo.version>3.0.0</dubbo.version>
+        <dubbo.version>3.0.3-SNAPSHOT</dubbo.version>
```
Comment (AlbumenJ) [inline dubbo-samples-triple/pom.xml:34]: please use relased version in samples

**Comment by x-shadow-man** [comment]
@guohao hi，pls assign this test to me. I'll do it.

File: dubbo-samples-triple/case-versions.conf [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+# Supported component versions of the test case
+
+# Spring app
+dubbo.version=3.0.3-SNAPSHOT
```
Comment (plusmancn) [inline dubbo-samples-triple/case-versions.conf:22]: Plz change `3.0.3-SNAPSHOT` to `3.*`

File: test/run-tests.sh [hunk 3]
```diff
@@ -112,6 +112,9 @@ function process_case() {
   case_dir=$1
   case_no=$2
 
+  echo "case_dir=$case_dir"
+  echo "case_no=$case_no"
+
```
Comment (plusmancn) [inline test/run-tests.sh:117]: Plz clean the debug code.

**Pull request merged**